### PR TITLE
Experimental support for generating png from html

### PR DESCRIPTION
**Pull request by FroMage**

### CODE REVIEW

**Comment by Karm** [comment]
See example on https://github.com/Karm/dev-null/tree/main/quarkus-openhtmltopdf-java2d

**Comment by FroMage** [comment]
But… this is without any special build item? Just using `quarkus-awt`?

**Comment by Karm** [comment]
If we were to create an **extension** covering openhtml suite of libraries, then there would be buildItems.

File: pdf/pom.xml [hunk 1]
```diff
@@ -13,17 +13,28 @@
...
+    <!-- 
+    <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-awt</artifactId>
+    </dependency>
+    -->
   	<dependency>
-  		<groupId>com.openhtmltopdf</groupId>
+  		<groupId>io.github.openhtmltopdf</groupId>
   		<artifactId>openhtmltopdf-core</artifactId>
```
Comment (Karm) [inline pdf/pom.xml:24]: @FroMage You don't need that. java2d pulls that in.

**Pull request merged**